src/lib/Behat/Component/UniversalDiscoveryWidget.php

```diff
@@ -125,6 +159,8 @@ protected function specifyLocators(): array
             new CSSLocator('cancelButton', '.c-top-menu__cancel-btn'),
             new CSSLocator('mainWindow', '.m-ud'),
             new CSSLocator('selectedLocationsTab', '.c-selected-locations'),
+            new CSSLocator('categoryTabSelector', '[data-bs-original-title="%s"]'),
+            new CSSLocator('selectedTab', '.c-tab-selector__item--selected, [title="%s"]'),
```

Comment:
This one looks like a mistake:
`,` is an alternative, so I will match elements that meet at least one of criteria:
1) has `c-tab-selector__item--selected` class
2) has `title` attribute equals to the value given

I'd use just `.c-tab-selector__item--selected` here